libs/core/src/scss/base/_variables.scss

```
@@ -119,7 +119,12 @@ $avatar-badge-size: 16px;
 
 /* Page
 ****************************************************************************/
-$page-content-max-width: 720px;
+$page-content-max-widths: (
+  default: 792px,
+  optimized: 1092px,
+  full: 100%,
+);
+$page-content-max-width: map.get($page-content-max-widths, 'default');
```

Consider adding a new variable for simpler usage, one that doesn't get deprecated/removed later:
```suggestion
$page-content-max-width: map.get($page-content-max-widths, 'default');
$page-content-max-width-default: map.get($page-content-max-widths, 'default');
```